docs/docs/actions.md

```diff
@@ -1,18 +1,18 @@
 ---
 title: Actions
-description: Documentation on actions and how they help you manipulate state within Gatsby
+description: Dokumentation für Actions und wie diese helfen State in Gatsby zu manipulieren
 jsdoc:
   - "gatsby/src/redux/actions/public.js"
   - "gatsby/src/redux/actions/restricted.js"
 contentsHeading: Functions
 ---
 
-Gatsby uses [Redux](http://redux.js.org) internally to manage state. When you implement a Gatsby API, you are passed a collection of actions (equivalent to actions bound with [bindActionCreators](https://redux.js.org/api/bindactioncreators/) in Redux) which you can use to manipulate state on your site.
+Gatsby verwendet [Redux](http://redux.js.org) unter der Haube um State zu verwalten. Wenn du eine Gatsby API verwendest bekommst du eine Auswahl von Actions (ähnlich der [bindActionCreators](https://redux.js.org/api/bindactioncreators/) Actions in Redux), welche du verwenden kannst um State in deiner App zu verwalten.
```

Comment:
```suggestion
Intern verwendet Gatsby [Redux](http://redux.js.org) um State zu verwalten. Wenn du eine Gatsby API verwendest, bekommst du eine Auswahl von Actions (ähnlich der [bindActionCreators](https://redux.js.org/api/bindactioncreators/) Actions in Redux), welche du dann zum verwalten von State in deiner App verwenden kannst.
```

The main change here would be to remove "unter der Haube". I like it a lot, but I believe it's a bit too sloppy :100:

Reply:
Agree with you, happy with your change!